README.md

```diff
@@ -1,4 +1,47 @@
 metro
 =====
 
-Module for sending metric to configurable pipelines
+Node module for sending metrics to graphite time series DB
+
+How to use metro in your repo:
+==============================
+
+1. `npm install metro -s`
+
+2. As part of the setup process create factory for graphite pipeline. The recommended way to do this is in a dedicated file. E.g:
+
+    const config = require('config');
+    const metro = require('metro');
+
+    module.exports = function setupMetro() {
+
+      let metroClient = null;
+      metro.create('default');
```

Comment:
perhaps create `latency_route` here as well, since it's used in the example belowz